### PR TITLE
DEVX-3566 render active css class for static pages

### PR DESCRIPTION
**Pull request by bencgreenberg**
Add CSS active class to static landing pages

### CODE REVIEW

File: lib/nexmo_developer/app/controllers/static_controller.rb [hunk 1]
```diff
@@ -45,6 +45,7 @@ def default_landing
       migrate_tropo
     end
 
+    @navigation = request.path.split('/').last.to_sym
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/controllers/static_controller.rb:48]: you can use `@navigation = yaml_name` here instead

File: lib/nexmo_developer/app/presenters/topnav_item.rb [hunk 2]
```diff
@@ -13,7 +13,7 @@ def title
 
   def css_classes
     classes = ['Vlt-tabs__link']
-    classes << 'Vlt-tabs__link_active' if @navigation == @name.downcase
+    classes << 'Vlt-tabs__link_active' if @navigation == @name.downcase || @navigation == @url.delete('/')
```
Comment (fabianrbz) [inline lib/nexmo_developer/app/presenters/topnav_item.rb:16]: I'm not sure about this, I would change `@name` in the config file to match the landing page file name (and the keys in the locale files too). that way we keep everything consistent.
WDYT?

File: lib/nexmo_developer/app/controllers/static_controller.rb (hunk 1 above)
Comment (bencgreenberg) [reply lib/nexmo_developer/app/controllers/static_controller.rb:48]: 👍

File: lib/nexmo_developer/app/presenters/topnav_item.rb (hunk 2 above)
Comment (bencgreenberg) [reply lib/nexmo_developer/app/presenters/topnav_item.rb]: Done. PR opened in NDP to update the config files as well: https://github.com/Nexmo/nexmo-developer/pull/3043

**Pull request merged**